tests/tests_suite_test.go

```diff
@@ -15,9 +17,10 @@ func TestTests(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	tests.BeforeTestSuitSetup()
+	tests.LookupBinaryCommand()
```

Comment:
I don't think it should be here. If it fails, it prevent other tests to run.

Reply:
No, because it shouldn't be failed. and this function is bump from upstream since it removed it.

Reply:
Why it shouldn't be failed?

Reply:
Take a look at `sanity_test.go`, it does not use any of these binaries. If this check failed, it prevent that tests to be run. And not all of them are required for a single tests, like `kubectl` is not necessary for most of the case.

Reply:
I remove this function `LookupBinaryCommand` already